lib/galaxy/datatypes/genetics.py

```diff
@@ -827,6 +829,462 @@ def __init__(self, **kwd):
                                 substitute_name_with_metadata='base_name', is_binary=True)
 
 
+class LinkageStudies(Text):
+    """
+    superclass for classical linkage analysis suites
+    """
+    test_files = [
+        'linkstudies.allegro_fparam', 'linkstudies.allegro_ihaplo',
+        'linkstudies.alohomora_gts', 'linkstudies.linkage_pedin',
+        'linkstudies.linkage_datain', 'linkstudies.linkage_map'
+    ]
+
+    def __init__(self, **kwd):
+        Text.__init__(self, **kwd)
+        self.lcount = 0
+        self.max_lines = 10
+        # iterate whole file without errors
+        self.eof_res = True
+
+    def tokenizer(self, line, sep=None):
```

Comment:
This function becomes now obsolete, right? It is exactly what split() is doing.

Reply:
I initially sought to standardize, but yes it's likely extra overhead now